dev/NavigationView/NavigationView.cpp

```diff
@@ -3496,6 +3496,7 @@ void NavigationView::OnLoaded(winrt::IInspectable const& sender, winrt::RoutedEv
         m_titleBarMetricsChangedRevoker = coreTitleBar.LayoutMetricsChanged(winrt::auto_revoke, { this, &NavigationView::OnTitleBarMetricsChanged });
         m_titleBarIsVisibleChangedRevoker = coreTitleBar.IsVisibleChanged(winrt::auto_revoke, { this, &NavigationView::OnTitleBarIsVisibleChanged });
     }
+    UpdatePaneButtonsWidths();
```

Comment:
nit: maybe adding a comment here to explain why this needs to be called in OnLoaded will be helpful

Reply:
Good idea, done.